packages/angularjs-extensions/README.md

```diff
@@ -0,0 +1,35 @@
+# RainCatcher Angular.js Extensions
```

Comment:
ping @austincunningham 

Patterns here may be useful for your documentation. 
This was really simple and covers all areas.